packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -127,6 +127,12 @@ abstract class RunCommandBase extends FlutterCommand with DeviceBasedDevelopment
               'this comma separated list of allowed prefixes.',
         valueHelp: 'foo,bar',
       )
+      ..addOption('trace-skia-allowlist',
+        hide: !verboseHelp,
+        help: 'Filters out all Skia trace events except those that are specified in '
+              'this comma separated list of allowed prefixes.',
+        valueHelp: 'foo,bar',
```

Comment:
when using this flag to expose traces related to shader compilation, what are the values I'll want to pass in here?

Reply:
Skia is using the tag "skia.shaders" for those events.

Changed the valueHelp to list some of the Skia tags.